src/Symfony/Component/Mailer/Exception/HttpTransportException.php

```diff
@@ -11,9 +11,24 @@
 
 namespace Symfony\Component\Mailer\Exception;
 
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
 class HttpTransportException extends TransportException
 {
+    private $response;
+
+    public function __construct(string $message = null, ResponseInterface $response, int $code = 0, \Exception $previous = null)
```

Comment:
do we ever use $previous? should we remove it if not?

Reply:
They allow future usage. I'd like to keep them.